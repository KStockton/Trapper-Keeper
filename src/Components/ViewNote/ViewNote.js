import React from 'react';
import { Link } from 'react-router-dom';

const ViewNote = ({ id, title }) => {
  return (
    <div>
      <Link to={`/`} className='back-btn'>â—€ back</Link>
      <h1>{title}</h1>
      <p>kjsdfnkasjfkasjndfkasjdnfkasjdnfksjdnfsdnc</p>
    </div>
  )
}

export default ViewNote;
