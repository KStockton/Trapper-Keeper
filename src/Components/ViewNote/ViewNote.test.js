import React from 'react';
import { shallow } from 'enzyme';
import ViewNote from './ViewNote'


describe('ViewNote', () => {
    let wrapper;
    beforeEach(() => {
        wrapper = shallow(<ViewNote/>)
    })
    it('should match snapshot', () => {
    expect(wrapper).toMatchSnapshot()
     
    })
})