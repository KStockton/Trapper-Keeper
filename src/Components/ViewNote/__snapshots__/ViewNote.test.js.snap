// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ViewNote should match snapshot 1`] = `ShallowWrapper {}`;
